### PR TITLE
Add sharp aquos doc

### PR DESCRIPTION
**Pull request by titilambert**
**Description:**

Hello !
This component give control on Sharp Aquos TVs

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4679

### CODE REVIEW

File: source/_components/media_player.aquostv.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - platform: aquostv
+    host: 192.168.0.10
+    port: 10002
+    username: admin
+    password: password
+```
+
+Configuration variables:
+
+- **host** (*Required*): The IP of the Samsung Smart TV, eg. `192.168.0.10^.
```
Comment (Landrash) [inline source/_components/media_player.aquostv.markdown]: Extra `^`. Host is generally IP and hostname.

**Comment by titilambert** [comment]
Hello !
I made changes. Tell me if this is good for you.

Thanks

**Comment by Landrash** [comment]
Looks good. One further enhancement that can be is to minimize the example since they are often cut and pasted. Optional variables do not need to be in the example for example.

**Comment by titilambert** [comment]
I just minimize the example as suggested

**Comment by Landrash** [comment]
Looks good @titilambert .
This can be merged when parent pr is merged.

**Comment by mattthhdp** [comment]
Dont know if its my Sharp tv, but Everything is working perfectly exept Turn_on, who is not working, if you need any more info that i am not aware of make a sign

**Pull request merged**